snaps/build-and-release-aws-eks-snaps.sh

```diff
@@ -4,13 +4,12 @@
 # This script will build and release snaps for AWS EKS.
 set -eux
 
-EKS_RELEASE="${EKS_RELEASE:-edge/eks.0}"
+EKS_RELEASE="${EKS_RELEASE:-edge}"
 KUBE_VERSION="${KUBE_VERSION:-$(curl -L https://dl.k8s.io/release/stable.txt)}"
 KUBE_ARCH="amd64"
+export SNAP_SUFFIX="eks"
```

Comment:
unused?

Reply:
Nah, it's used -- just not here.  We need it in the env to be picked up by the build script here:

https://github.com/juju-solutions/release/blob/rye/snaps/snap/build-scripts/build#L46

I'll add a comment to that effect.